contracts/upgradeable_contracts/U_ForeignBridge.sol

```diff
@@ -215,11 +234,20 @@ contract ForeignBridge is ERC677Receiver, BasicBridge {
         emit DailyLimit(_foreignDailyLimit);
     }
 
+    function setHomeDailyLimit(uint256 _homeDailyLimit) external onlyOwner {
+        uintStorage[keccak256("homeDailyLimit")] = _homeDailyLimit;
+    }
+
     function withinLimit(uint256 _amount) public view returns(bool) {
         uint256 nextLimit = totalSpentPerDay(getCurrentDay()).add(_amount);
         return foreignDailyLimit() >= nextLimit && _amount <= maxPerTx() && _amount >= minPerTx();
     }
 
+    function withinHomeLimit(uint256 _amount) public view returns(bool) {
+        uint256 nextLimit = totalSpentPerDay(getCurrentDay()).add(_amount);
```

Comment:
the variable `totalSpentPerDay` is logically linked with the daily limit related to the operations on the foreign network. So, this check is not correct. Please introduce another structure to track the daily limit related to operations on the home network.

Reply:
Thanks, Fixed.